hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/ZKRMStateStore.java

```diff
@@ -1441,6 +1441,29 @@ void delete(final String path) throws Exception {
     zkManager.delete(path);
   }
 
+  /**
+   * Deletes the path more safe.
+   * When NNE is encountered, if the node does not exist,
```

Comment:
Could you expand NNE in the javadoc for brevity?

Reply:
> Could you expand NNE in the javadoc for brevity?

Okay, thank you for your reminder